modules/structs/org_type.go

```diff
@@ -40,6 +40,16 @@ func (vt VisibleType) IsPrivate() bool {
 	return vt == VisibleTypePrivate
 }
 
+// VisibilityString provides the mode string of the visibility type (public, limited, private)
+func (vt VisibleType) VisibilityString() string {
```

Comment:
Why not just use `String` that would implemented `fmt.Stringer` interface?
```suggestion
func (vt VisibleType) String() string {
```